Dockerfile

```diff
@@ -132,8 +135,8 @@ EXPOSE 80 443
 
 LABEL name="manageiq" \
           vendor="ManageIQ" \
-          version="Capablanca" \
-          release="latest" \
+          version="Master" \
```

Comment:
I want to use "Master" on the master branch and "Darga" on the darga branch so that people don't accidentally confuse the two.